src/templates/language_selector.html

```diff
@@ -1,4 +1,4 @@
-<ul>
+<div id="language-selector">
```

Comment:
https://github.com/dimagi/Vellum/commit/d514f2fe0cfad864746fcf36e511dfa4527f0665

> Only append the language selector if it isn't already there (add a flag id/class to the divider or the nav-header)

Couldn't do that since technically you could add/remove a translation in source xml (although formbuilder might strip that out since it is passed in a list of valid languages)